packages/react-components/src/Popup.tsx

```diff
@@ -2,6 +2,30 @@
 // This software may be modified and distributed under the terms
 // of the Apache-2.0 license. See the LICENSE file for details.
 
-import { Popup } from 'semantic-ui-react';
+import { Popup as PopupBase, StrictPopupProps } from 'semantic-ui-react';
+import React from 'react';
+import styled from 'styled-components';
 
-export default Popup;
+function Popup({ content, trigger, className }: StrictPopupProps): React.ReactElement {
+    return <PopupBase
+      content={content}
+      trigger={trigger}
+      className={className}
+    />
+}
+
+export default styled(Popup)`
+    background-color: #1B1C1D !important;
+    ::before {
+        background-color: #1B1C1D !important;
+    }
+
+    .ui--popUpText, p {
+        color: #FFFFFF;
```

Comment:
To get theming work, like `changing one line of code to apply the new colour applied in all components`, we need to centralise colour definition in [a colour-definition file](https://github.com/cennznet/ui/blob/develop/styled-theming/themes/dark/colors.ts).

To get those colours applied in components, it can be refactored as

```js
import { colors } from '../path/to/styled-theming';
...
color: ${colors.N0}; // N for Nuetral here, and more details could be found with design team
...
```

Reply:
Similar approach would apply to other similar code in this PR

Reply:
Not sure how the `B1` N1` scale thing works but added a new const there